docs/usage/interpreter/installation.md

```diff
@@ -47,31 +47,6 @@ You can get full list of community managed interpreters by running
 ./bin/install-interpreter.sh --list
 ```
 
-#### Install interpreter built with Scala 2.10
-Zeppelin support both Scala 2.10 and 2.11 for several interpreters as below:
-
-<table class="table-configuration">
-  <tr>
-    <th>Name</th>
-    <th>Maven Artifact for Scala 2.10</th>
-    <th>Maven Artifact for Scala 2.11</th>
-  </tr>
-  <tr>
-    <td>spark</td>
-    <td>org.apache.zeppelin:zeppelin-spark_2.10:0.10.0</td>
-    <td>org.apache.zeppelin:zeppelin-spark_2.11:0.10.0</td>
-  </tr>
-</table>
-
-#### Install Spark interpreter built with Scala 2.10
```

Comment:
it's outdated. https://mvnrepository.com/artifact/org.apache.zeppelin/zeppelin-spark